pages/common/qutebrowser.md

```diff
@@ -0,0 +1,24 @@
+# qutebrowser
+
+> A keyboard-driven, vim-like browser based on PyQt5.
+> More information: <https://qutebrowser.org/>.
+
+- Open qutebrowser with a specified (non-default) storage directory:
+
+`qutebrowser --basedir {{base_directory}}`
```

Comment:
"non-default" doesn't seem necessary to add here. Is it?
```suggestion
- Open qutebrowser with a specified storage directory:

`qutebrowser --basedir {{path/to/directory}}`
```
Also, `base_directory` in the variable name doesn't seem to give it much more value than with omitting it 🤔